src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,4 +1,5 @@
 *REMOVED*abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.MemberAccessExpression(Microsoft.CodeAnalysis.SyntaxNode expression, Microsoft.CodeAnalysis.SyntaxNode memberName) -> Microsoft.CodeAnalysis.SyntaxNode
+*REMOVED*Microsoft.CodeAnalysis.Workspace.RaiseDocumentActiveContextChangedEventAsync(Microsoft.CodeAnalysis.Document document) -> System.Threading.Tasks.Task
```

Comment:
The API must be kept.  If you feel strongly about this the path forward is to keep it + mark it as obsolete (with an error).